### DISCUSSION TITLE
Метод API для получения данных подписей обеих сторон

### DISCUSSION THREAD

**Discussion by IamrrrED** [diadoc/diadocsdk-1c-docs#963] [Q&A] [answered]
Здравствуйте.

Прошу подсказать метод API, который я могу использовать для получения данных подписей обеих сторон подписанного документа. В общем случае мне нужны организации, указанные в подписях.
Благодарю.

**Comment by GilimkhanovDenis** [comment, marked as answer]
https://github.com/diadoc/diadocsdk-1c-docs/issues/964 создал issue, ответил в нём:

осмотрите на методы объекта [DocumentBase](https://developer.kontur.ru/Docs/Diadoc/COM/ComObjects/DocumentBase.html) во вкладке Информация о контентах и подписях

**Reply by IamrrrED** [reply]
Пример получения данных для 1С указанным методом можете подсказать?

**Reply by GilimkhanovDenis** [reply]
Не очень понял, что именно хотите увидеть, помимо того, что написано в документации:
`dd_Signature = DocumentBase.GetSenderSignature();`

**Reply by IamrrrED** [reply]
Параметры для вызова DocumentBase. Как из объекта вызвать метод GetSenderSignature()?

**Reply by GilimkhanovDenis** [reply]
у GetSenderSignature нет параметров

**Reply by IamrrrED** [reply]
Как из 1С обратиться к DocumentBase? Именно тот документ, который необходим мне.

**Reply by GilimkhanovDenis** [reply]
Предлагаю обратиться к разделу [Как работать с документами](https://developer.kontur.ru/Docs/Diadoc/COM/HowTo/HowTo_document.html)
